python_modules/dagster/dagster/_core/definitions/utils.py

```diff
@@ -96,6 +96,14 @@ def validate_tags(
 ) -> Mapping[str, str]:
     valid_tags: Dict[str, str] = {}
     for key, value in check.opt_mapping_param(tags, "tags", key_type=str).items():
+        if key == PRIORITY_TAG:
+            try:
+                int(value)
```

Comment:
how explicit are we that these should be `int` instead of `float` ? It looks like strings-of-floats don't coerce to `int` even though actual floats do:

```
>>> int(100.5)
100
>>> int("100.5")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '100.5'
```

does the current set-up silently treat floats as bad priorities?